docs/questions/native-editor/sql-parameters.md

```diff
@@ -139,14 +139,20 @@ In the variables sidebar, you can set a default value for your variable. This va
 
 ## Setting complex default values in the query
 
-You can also define default values directly in your query, which is useful when defining complex default values (for example, using a function to generate a value).
+You can also define default values directly in your query by enclosing a comment inside the end brackets of an optional parameter:
 
-Here's an example that sets the default value of a Date filter to the current date using `CURRENT_DATE()`:
+```
+WHERE column = [[ {% raw %}{{ your_parameter }}{% endraw %} #]]your_default_value
```

Comment:
In the future, we should probably wrap whole code blocks in raw tags, if only to make the code easier to read in reviews and in the markdown files.